VirtoCommerce.Platform.Core/Common/TenantIdentity.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace VirtoCommerce.Platform.Core.Common
+{
+    public class TenantIdentity : ValueObject<TenantIdentity>
+    {
```

Comment:
Need to add ctor TenantIdentity(string tenantId, string tenantType) and Override ToString method which will return glued TenantId + "_" + TenantType

Reply:
somehow I was expecting to learn about performance, execution plans and indices from you in here. But you're dead simple:)